frameworks/bitcoin-move/sources/bbn.move

```diff
@@ -57,12 +60,32 @@ module bitcoin_move::bbn {
         staking_time: u16
     }
 
+    struct BBNStakeSeal has key {
```

Comment:
Should add `store` to make it transferable?

Reply:
BBNStakeSeel is L1 asset like Inscription